Header header
TactSensor[16] sensorArray
